index.js

```diff
@@ -603,13 +528,11 @@ BFFS.prototype.promote = function promote(spec, callback) {
       });
     }))
     .on('error', fn)
-    .on('data', (data) => {
-      operations.push((next) => {
-        async.parallel([
-          (cb) => BuildHead.create(this._strip(data), cb),
-          changed && ((cb) => Build.update(data, cb))
-        ].filter(Boolean), next);
-      });
+    .on('data', data => {
+      operations.push(next => async.series([
+        BuildHead.update.bind(BuildHead, this._strip(data)),
```

Comment:
Note this is now `update` rather than `create`, should be fine see description

Reply:
does `update` work if its the first time a build head was created?

Reply:
See description?

Reply:
I apparently fail at reading today, both your description and your actual comment.
![fail](https://media.giphy.com/media/PUtqq38UYoI3m/giphy.gif)